src/@next/Select/Select.tsx

```diff
@@ -43,24 +42,54 @@ export interface SelectProps {
 
 export const Select = ({
   allowMultiple = false,
-  disabled,
-  hasError,
+  disabled = false,
+  hasError = false,
   helpText,
   label,
   onClose,
   onRemoveTag,
   onSelect,
-  options,
+  options = [],
   placeholder,
   listHeight,
+  searchable = false,
   searchableProps,
-  scrollable,
+  scrollable = false,
   sections,
   selectedValues,
   width,
 }: SelectProps) => {
   const [popoverActive, setPopoverActive] = useState(false);
   const [optionListHeight, setOptionListHeight] = useState('');
+  const [menuOptions, setMenuOptions] = useState(options);
+  const [inputValue, setInputValue] = useState(
```

Comment:
will this work if user programatically changed the value?

Reply:
it won't work but we'll be removing this in a separate PR after this one. to handle the input state internally. currently I just added a TODO comment beside the `searchableProps` prop and we'll help to check and replace if it's already been used in employers.

the current way for searchable multi select to update input value is to also pass `searchableProps?.onInputChange`. we also already have it on the storybook code docs